modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -29,6 +29,42 @@
 #include <cstdlib>
 
 namespace cv {
+#if CV_SIMD
+#if CV_AVX512_SKX
+    template<typename T>
+    CV_ALWAYS_INLINE v_float32x16 v_setr(const T* ptr)
+    {
+        return v_float32x16(_mm512_setr_ps(*ptr, *(ptr + 1), *(ptr + 2), *(ptr + 3),
+            *(ptr + 4), *(ptr + 5), *(ptr + 6), *(ptr + 7),
+            *(ptr + 8), *(ptr + 9), *(ptr + 10), *(ptr + 11),
+            *(ptr + 12), *(ptr + 13), *(ptr + 14), *(ptr + 15)));
```

Comment:
Why do we need this? Why does `v_load()` not work?

Reply:
@alalek This snippet is on internal review. So it's refactoring now. I'll let you know when it becomes ready for your final review. Sorry for inconvenient.

Reply:
OK, converted to "draft" (just UI marker, no changes in automation are expected).

Reply:
> Why do we need this? Why does `v_load()` not work?

Outdated.